src/py42/modules/detectionlists.py

```diff
@@ -86,12 +87,13 @@ def add_user_risk_tags(self, user_id, tags):
         self._detection_list_user_client.add_risk_tags(user_id, tags)
 
     def remove_user_risk_tags(self, user_id, tags):
-        """Remove one or more tags.
+        """Remove one or more risk factors.
```

Comment:
@alanag13 Is arguing for us to all these `tags` last I checked, believing that `factors` is a marketing term.

Reply:
I know Alan is looking to use tags as a more generic term, but my thought was to tie it back to risk factors in the explanation here, since both the API and UI use "risk factors".

Reply:
Should we say "risk factor tags"?

Reply:
That seems like a good compromise to me.

Reply:
documentation is more editable than function names -- im fine with docs containing whatever info will best be able to help the user figure out what's going on.